src/lib/containers/UserCardSmall.tsx

```diff
@@ -31,19 +32,21 @@ export const UserCardSmall: React.SFC<UserCardSmallProps> = (
       profileClickHandler(userProfile)
     }
   )
-  if (userProfile.displayName) {
-    label += userProfile.displayName
-  } else if (userProfile.firstName && userProfile.lastName) {
-    label += (`${userProfile.firstName} ${userProfile.lastName}`)
-  }
-  if (userProfile.userName && !userProfile.userName.includes('TEMPORARY-')) {
-    label += ` (${userProfile.userName})`
-  }
-  if (userProfile.position) {
-    label += ` <br/>${userProfile.position}`
-  }
-  if (userProfile.location) {
-    label += ` <br/>${userProfile.location}`
+  if (!hideTooltip) {
```

Comment:
we could do a conditional render on ReactToolTip using the hideToolTip variable, e.g. {
   !hideToolTip && <ReactToolTip..../>
} instead of putting that on the label

Reply:
Yes, I tried that.  It still showed the tooltip!

Reply:
ohh funky, just to clarify, my comment would not use the disable prop on the ReactTooltip, it would be a conditional render on the whole Reactooltip itself, that did not work?

Reply:
Understood, and that's what I initially tried (mimicking conditional adding of span using hideText), but was surprised to still see the tooltip.

Reply:
Gotcha, okay, well then all looks good hear, ill merge.